MoltenVK/MoltenVK/Layers/MVKExtensions.def

```diff
@@ -41,102 +41,103 @@
+MVK_EXTENSION(KHR_vulkan_memory_model,                KHR_VULKAN_MEMORY_MODEL,                DEVICE,   MVK_NA, MVK_NA)
+MVK_EXTENSION(EXT_buffer_device_address,              EXT_BUFFER_DEVICE_ADDRESS,              DEVICE,   13.0,  16.0)
+MVK_EXTENSION(EXT_debug_marker,                       EXT_DEBUG_MARKER,                       DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_debug_report,                       EXT_DEBUG_REPORT,                       INSTANCE, 10.11,  8.0)
+MVK_EXTENSION(EXT_debug_utils,                        EXT_DEBUG_UTILS,                        INSTANCE, 10.11,  8.0)
+MVK_EXTENSION(EXT_descriptor_indexing,                EXT_DESCRIPTOR_INDEXING,                DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_external_memory_host,               EXT_EXTERNAL_MEMORY_HOST,               DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_fragment_shader_interlock,          EXT_FRAGMENT_SHADER_INTERLOCK,          DEVICE,   10.13, 11.0)
+MVK_EXTENSION(EXT_hdr_metadata,                       EXT_HDR_METADATA,                       DEVICE,   10.15, MVK_NA)
+MVK_EXTENSION(EXT_host_query_reset,                   EXT_HOST_QUERY_RESET,                   DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_image_robustness,                   EXT_IMAGE_ROBUSTNESS,                   DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_inline_uniform_block,               EXT_INLINE_UNIFORM_BLOCK,               DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_memory_budget,                      EXT_MEMORY_BUDGET,                      DEVICE,   10.13, 11.0)
+MVK_EXTENSION(EXT_metal_objects,                      EXT_METAL_OBJECTS,                      DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_metal_surface,                      EXT_METAL_SURFACE,                      INSTANCE, 10.11,  8.0)
+MVK_EXTENSION(EXT_pipeline_creation_cache_control,    EXT_PIPELINE_CREATION_CACHE_CONTROL,    DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_post_depth_coverage,                EXT_POST_DEPTH_COVERAGE,                DEVICE,   11.0,  11.0)
+MVK_EXTENSION(EXT_private_data,                       EXT_PRIVATE_DATA,                       DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_robustness2,                        EXT_ROBUSTNESS_2,                       DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_sample_locations,                   EXT_SAMPLE_LOCATIONS,                   DEVICE,   10.13, 11.0)
+MVK_EXTENSION(EXT_sampler_filter_minmax,              EXT_SAMPLER_FILTER_MINMAX,              DEVICE,   MVK_NA, MVK_NA)
+MVK_EXTENSION(EXT_scalar_block_layout,                EXT_SCALAR_BLOCK_LAYOUT,                DEVICE,   10.11,  8.0)
+MVK_EXTENSION(EXT_shader_atomic_float,                EXT_SHADER_ATOMIC_FLOAT,                DEVICE,   13.0,  16.0)
+MVK_EXTENSION(EXT_shader_demote_to_helper_invocation, EXT_SHADER_DEMOTE_TO_HELPER_INVOCATION, DEVICE,   11.0,  14.0)
+MVK_EXTENSION(EXT_separate_stencil_usage,             EXT_SEPARATE_STENCIL_USAGE,             DEVICE,   10.11,  8.0)
```

Comment:
Diff of this file looks okay. However, this item has ended up out of alpha order. Move this up two lines to above all `EX_shader*` entries.